lendingbot.py

```diff
@@ -344,9 +345,9 @@ def cancelAndLoanAll():
 		activeCurIndex += 1
 		activeCurTestBalance = Decimal(lendingBalances[activeCur])
 		activeBal = 0
-                if activeCur in totalLended:
-                	activeCurTestBalance += Decimal(totalLended[activeCur])
-		
+		if activeCur in totalLended:
+			activeCurTestBalance += Decimal(totalLended[activeCur])
```

Comment:
On this one I fixed the lines being unnecessarily indented. How it runs has not been changed